src/index.html

```diff
@@ -7,8 +7,58 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header-logo" href="http://127.0.0.1:5500/src/index.html">
```

Comment:
Why here such href?
It is follow to not existance page

Reply:
cuz I don't how specify the correct way to the home page, because it is not there yet...

Reply:
You can apply just a placeholder `#`

Reply:
I did so. i wrote #home